code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -60,6 +60,9 @@
 	return TRUE
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
+	if(istype(A, /obj/item/reagent_containers/syringe/bluespace))
+		to_chat(user, "<span class='notice'>\The [A] is too big to load into \the [src].</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>[A] is too big to load into [src].</span>")
```

"The" is automatic.

Reply:
Don't you tell me that the The is automatic. I'll cry.